kubernetes/samples/scripts/domain-lifecycle/scaleCluster.sh

```diff
@@ -0,0 +1,123 @@
+# !/bin/sh
+# Copyright (c) 2020, 2021, Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+#
+
+script="${BASH_SOURCE[0]}"
+scriptDir="$( cd "$( dirname "${script}" )" && pwd )"
+source ${scriptDir}/helper.sh
+if [ "${debug}" == "true" ]; then set -x; fi;
+
+function usage() {
+
+  cat << EOF
+
+  This script scales a WebLogic cluster in a domain by patching
+  'spec.clusters[<cluster-name>].replicas' attribute of the domain
+  resource. This change will cause the operator to perform a scaling
+  operation for the WebLogic cluster based on the value of replica count.
+ 
+  Usage:
+ 
+    $(basename $0) -c mycluster -r replicas [-n mynamespace] [-d mydomainuid] [-m kubecli]
+  
+    -c <cluster-name>   : Cluster name parameter is required.
```

Comment:
Can we consider defaulting this to 'cluster-1'? That's what we use in the samples...

Reply:
We don't default cluster-name to `cluster-1` for `stopCluster.sh` and `startCluster.sh` scripts. If there are multiple clusters in the domain and the user intended to change replicas for `cluster-2` but forgot to specify the cluster-name parameter then it might cause unintended behavior.

Reply:
Ok.